.github/ISSUE_TEMPLATE/help-request.yml

```diff
@@ -0,0 +1,11 @@
+name: Other questions
+description: For all other questions / issues
+labels: ["question"]
+body:
+  - type: textarea
+    id: question-body
+    attributes:
+      label: Describe your question.
+      description: Please provide any relevant information
+    validations:
+      required: true
```

Comment:
The wording should relate to issues and, more generally, requests (as in feature requests), but _not_ questions. The issue tracker is not a place for questions.

Reply:
I deleted this form entirely and added a link to the discussions where I had put the discord link